server/tinytuya

```diff
@@ -0,0 +1 @@
+../tinytuya
```

Comment:
Oops, this wasn't supposed to be committed...

Reply:
I've done the same, many times.  Easy fix!  Updated.